lib/utils/eject/modify-package-json.js

```diff
@@ -0,0 +1,57 @@
+const fs = require('fs-extra');
+const path = require('path');
+const spawn = require('cross-spawn');
+
+const appDependencies = require('../../app-dependencies');
+
+const writeJson = require('./write-json');
+
+const CWD = process.cwd();
+
+function filterDependencies(dependencies, test) {
+  return Object.keys(dependencies)
+  .filter(test)
+  .reduce((obj, key) => {
+    obj[key] = dependencies[key];
+    return obj;
+  }, {});
+}
+
+/**
+ * Moves (and upgrades) any SKY UX dependencies to the new Angular CLI package.json file.
+ */
+ async function modifyPackageJson(ejectedProjectPath) {
+  const packageJson = fs.readJsonSync(path.join(CWD, 'package.json'));
+  const ejectedPackageJson = fs.readJsonSync(path.join(ejectedProjectPath, 'package.json'));
+
+  packageJson.dependencies = packageJson.dependencies || {};
+  ejectedPackageJson.dependencies = ejectedPackageJson.dependencies || {};
+
+  const nonSkyuxSdkDependencies = filterDependencies(
+    packageJson.dependencies,
```

Comment:
Would there ever be `@skyux-sdk` dependencies in `dependencies`? Is this just a precaution?